docs/additional-functionality/shuffle-docker-examples/Dockerfile.ubuntu_rdma

```diff
@@ -35,7 +35,7 @@
 
 ARG RDMA_CORE_VERSION=32.1
 ARG CUDA_VER=11.8.0
```

Comment:
could we move CUDA_VER to 12.8? I am not positive all OS'es we are looking for are in 12.8, but it would be nice to try.